lnbits/app.py

```diff
@@ -213,10 +224,32 @@ async def basic_error(request: Request, err):
             content={"detail": err},
         )
 
+    @app.exception_handler(RequestValidationError)
+    async def validation_exception_handler(
+        request: Request, exc: RequestValidationError
+    ):
+        # Only the browser sends "text/html" request
+        # not fail proof, but everything else get's a JSON response
+
+        if (
+            request.headers
+            and "accept" in request.headers
+            and "text/html" in request.headers["accept"]
+        ):
+            return template_renderer().TemplateResponse(
+                "error.html",
+                {"request": request, "err": f"{exc.errors()} is not a valid UUID."},
```

Comment:
What does UUID refer to? Is this a special error or a general one for all request validation errors? If so, please adjust error message.

Reply:
It looks like `validation_exception_handler` was just moved from `L72`. This PR changed nothing about its content.

Reply:
right